packages/SwingSet/docs/virtual-objects.md

```diff
@@ -124,10 +122,10 @@ to obtain a new weak store instance, which you can then access using the `has`,
 
 However, the vat secondary storage system's `makeWeakStore` augments the one provided by the `@agoric/store` package in two important ways:
 
-- Weak store operations may use virtual object representatives as keys, and if you do this the corresponding underlying virtual object instance identities will be used for indexing rather than the object identities of the representative objects (i.e., two different representives for the same underlying virtual object will operate on the same weak store entry).
+- Weak store operations may use virtual objects as keys, and if you do this the corresponding underlying virtual object instance identities will be used for indexing rather than the current in-memory object identities of the objects.
```

Comment:
I think we can continue to call the current one-at-a-time in-memory javascript `Object` a "Representative", and use "virtual object" to refer to the longer-lived thing that has a vref and a stable identity. We might say that WeakSet/Map *methods* (`.get`, `.set`) use Representatives as keys, but the underlying abstract operations use the corresponding virtual object identity as a key.

Reply:
Since the one consequence (and arguably one of the purposes) of this set of changes is to make the representative abstraction effectively invisible to vat code, we should be careful how we talk about it in documentation.  To the extent that the docs are about implementation, talking about representatives makes sense, but to the extent they are use facing maybe not so much.  I'm a little unclear what the status of this particular document is in that regard, or if we've even thought about that.